### DISCUSSION TITLE
Got to the web interface once, but now I cannot

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#8519] [General] [answered]
First, I have to say version 2.3.140 is easy to install and the web interface came up the first time using my laptop chrome browser. This is where I got the installation instructions “https://www.youtube.com/watch?v=TG-pPm7KVro  “.
    I installed SO version 2.3.140. After 
    I am using :
    SO version 2.3.140.
    I am trying to use SO on my windows 10 laptop, I don’t use the cloud.
    I used securityonion-2.3.140-20220719.iso
    I used Centos 7
    For nodes, I just have laptops, TV’s and a printer.
    I am using a dell laptop and windows 10 version 10.0.19044 build 19044
    Not monitoring traffic right now until I get SO working.
    These are the services I choose.    
![security onion 2 2 140 a](https://user-images.githubusercontent.com/5569263/184159044-5b94a696-2331-4f4e-a9f0-e96a6ea2c4c7.png)

    so-status shows most ok
![security onion 2 2 140 b](https://user-images.githubusercontent.com/5569263/184159615-a6023a3f-3fc4-4c20-8c5c-ad77667f4719.png)

sudo salt-call state.highstate – this is what I got (now I am thinking that I need to reinstall and just select the eval service for now?)
![security onion 2 2 140 c](https://user-images.githubusercontent.com/5569263/184160170-10be7b25-f68b-4c8e-9288-d462b92ca3c0.png)

     My issue – I successfully installed SO 2.3.140, the sudo so-allow and the web interface came up using the suggested  IP URL. 
![security onion 2 2 140 d](https://user-images.githubusercontent.com/5569263/184160792-3024e25e-94e0-4607-ad69-125db940850e.png)

I used the suggested IP, and successfully got the web interface. 
![security onion 2 2 140 e](https://user-images.githubusercontent.com/5569263/184161724-9c4145a7-63ac-4e47-9512-71e73f5f966e.png)

But two days later the same https://192.168.224.130/ comes up with
![security onion 2 2 140 f](https://user-images.githubusercontent.com/5569263/184162208-d7619028-3210-4753-add9-09b55b51160c.png)

    I ran sudo soup. 
    When I ran sudo salt-call state.highstate, I saw this (I think this may be the problem)
![security onion 2 2 140 g](https://user-images.githubusercontent.com/5569263/184162850-5d6b3952-d7a6-4150-b3c8-04a115780d40.png)

  Please let me know if you need anything else, 
thanks for your advice and suggestions

**Comment by xfaith** [comment]
You could try manually starting alll dockers that are not started
so-elastalert-restasrt or so-elastalert-start.  

I would also look at 
docker logs so-elastalert and see if any errors are reported.

**Comment by iqworks** [comment]
David, thank you. I will give those a try as soon as I get back to this project.

**Comment by iqworks** [comment]
does all statuses need to be OK? Could this affect my connection to the web interface?

My main issue is that I cannot get a connection to the web interface. It worked once, but not now?

 

Thanks again for your help

**Reply by isaacgolding** [reply]
To answer your specific question all status need to be OK or things will not work right.     If you just installed and didn't tinker with it at all then your install or configuration was messed up somehow.

I'd start over with a fresh install and if you only have a couple of laptops and a few other internet devices you don't need all the extra services that you installed.   I would start over and make sure your machine (real or virtual) meets the min requirements found here:  

https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments

Then start over and perform a basic standalone install.

**Reply by iqworks** [reply]
thanks isaac, will do as soon as i get back to this project. 
thanks for the info

**Reply by iqworks** [reply]
Hi isaacgolding, I did as you suggested and reinstalled. Same thing happened, I could see the security onion the first time. But i noticed that when i did a restart, then i couldnt get the web interface again. same messages in the browser. 

this time I used securityonion-2.3.150-20220820.iso.  these are the changes in the install that i made : 
![image](https://user-images.githubusercontent.com/5569263/187297916-1fe065b8-cfa3-4d05-b6bc-3ed69fe5a6f0.png)

![image](https://user-images.githubusercontent.com/5569263/187298056-4dffdf1c-e9e9-4ecf-a530-dd4e91ca9a90.png)

![image](https://user-images.githubusercontent.com/5569263/187298096-7a93f8ab-316d-4a26-ba5a-d85066f5e503.png)

Call up chrome and enter “ https://192.168.80.130/ “
Select advanced
![image](https://user-images.githubusercontent.com/5569263/187298157-858c5787-a1f0-4f3e-9540-d59d0e8eb28d.png)

Click “proceed to 192.168.80.130”
![image](https://user-images.githubusercontent.com/5569263/187298304-ab78fdb6-dfd9-4b7e-9fa7-b23b1df5128c.png)

I get the login.
![image](https://user-images.githubusercontent.com/5569263/187298429-607eeb0a-4d8c-4725-b2d0-513207bac832.png)

This is what you get. 
![image](https://user-images.githubusercontent.com/5569263/187298510-834ec7b9-35a9-4a38-b4ee-f445a09d68d6.png)

Is it possible that one of my security onion install IP addresses gets changed when i do a restart?

thanks for your advice or suggestions.

**Reply by iqworks** [reply]
when i do a windows restart and try iq SO 22.04-desktop-amd64 MSEARCH 14 150  (securityonion-2.3.150-20220820.iso), this is what i get :  
![image](https://user-images.githubusercontent.com/5569263/187299947-3eb0feb3-5a7d-4ca3-bf3b-5e4d6cb113fe.png)

i think i need to look at the logs now (have to learn how to find and view them). 
but if there are any ideas or suggestions, please let me know.

**Reply by iqworks** [reply]
i see so many suggestions about how to find and view the logs. Which logs should i look at ?? 
thanks again.

**Comment by dougburks** [comment]
[As we've discussed previously](https://github.com/Security-Onion-Solutions/securityonion/discussions/5608), I would highly recommend that you stick with a simple IMPORT installation until you're fully comfortable with networking, virtualization, and logging.

If you follow the screenshots at https://docs.securityonion.net/en/2.3/first-time-users.html, then you'll have a graphical desktop and browser inside your VM and it might be easier to use that VM browser rather than trying to connect from a browser on your Windows host.

**Comment by iqworks** [comment]
Thanks Doug. The problem was I needed to start up the VM, then do so-status to see what has OK. I didn’t even have to wait for all of the dockers to show OK. Then I tried the url again and SO came up. I just understood that 5 minutes ago. 😊

**Comment by iqworks** [comment]
Thanks again Doug

 

Perhaps you could put those steps in your documentation? Some place in the installation guide that new people to security onion VM cannot miss. 

This issue took me weeks to fix.

**Comment by dougburks** [comment, marked as answer]
> Perhaps you could put those steps in your documentation? Some place in the installation guide that new people to security onion VM cannot miss.

Here's the relevant documentation:
https://docs.securityonion.net/en/2.3/post-installation.html#services
https://docs.securityonion.net/en/2.3/so-status.html#so-status
https://docs.securityonion.net/en/2.3/salt.html#salt-minion-startup-options

It shouldn't take that long for the web interface to initialize. If it's taking too long, that could mean that your VM is under-powered for the services that you've enabled. Thus my previous recommendations for you to run in a simple IMPORT installation.

**Comment by iqworks** [comment]
thanks

**Comment by iqworks** [comment]
Thanks again Doug for getting back. 

 

Ohhhhh, “If it's taking too long, that could mean that your VM is under-powered for the services that you've enabled.” Makes a lot of sense now. Like I said, I choose ALL the services for the VM I created using “securityonion-2.3.140-20220719.iso” first, but my last VM creation I used “securityonion-2.3.150-20220820.iso” was created without any services selected. 

 

I looked at these 3 links and I didn’t find these simple steps :

1 – power on the security onion (for the desktop version not inside the VM). 

2 – check the so-status and wait until everything is at OK (yep Doug, I did choose all of the tools the first time, but none this time).

3 – use your access IP address in a browser. 

 

I have two VMs. The first one has all services selected  - securityonion-2.3.140-20220719.iso



This caused my access url not to work



And the access url gets this



 

 

This is what I selected for the second one - securityonion-2.3.150-20220820.iso



This allowed my access url to work 



This is what I get 



 

 

Perhaps I am missing something in these three links?

 

No matter, things are working for now so I am good to go (I hope 😊), I hope this helps someone else.

**Comment by dougburks** [comment]
> I looked at these 3 links and I didn’t find these simple steps :
1 – power on the security onion (for the desktop version not inside the VM).

If you need documentation to tell you to power on Security Onion, then perhaps you still need to learn more about virtualization and networking as we've discussed previously:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5134
https://github.com/Security-Onion-Solutions/securityonion/discussions/5131
https://github.com/Security-Onion-Solutions/securityonion/discussions/5652

**Comment by iqworks** [comment]
Thanks Doug.
